spock-specs/src/test/groovy/org/spockframework/docs/datadriven/DataSpec.groovy

```diff
@@ -98,6 +100,99 @@ class DataSpec extends EmbeddedSpecification {
 // end::multiple-tables-with-top-border[]
   }
 
+  def "multiple data provider combined"() {
+    given:
+    def expected = '''
+      |// tag::data-providers-combined-result[]
```

Comment:
... no, definitely not.
It looked like expected with `stripIndent` but then the test failed on newer Groovy versions because of the Java stripIndent and I changed to `stripMargin` in the test, which then broke the doc, good find, thanks.

I'll look into it.

Actually, this is also the first place that violates the new auto-source-linker presumption that all such imports are within a literal or listing block.

But I wonder why it even then does not fail but misbehaves, I'll investigate.

Reply:
Now it should be as intended.